### PR TITLE
docs: add 1.11 release blog post

### LINKED ISSUE #7428: Release Notes for 1.11.0
### Is your enhancement related to a problem? Please describe

Create Release Notes for 1.11.0

### Describe the solution you'd like

Do release notes

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cdrage**
docs: add 1.11 release blog post

### What does this PR do?

Adds the 1.11 release blog post

### Screenshot / video of UI

N/A, see markdown

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop/issues/7428

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: website/blog/2024-06-24-release-1.11.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+---
+title: Podman Desktop 1.11 Release
+description: Podman Desktop 1.11 has been released!
+slug: podman-desktop-release-1.11
+authors: cdrage
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.11/light-mode.png
```
Comment (deboer-tim) [inline website/blog/2024-06-24-release-1.11.md]: IMHO this should be banner.png, and a second copy it needs to go in website/static/img/blog/podman-desktop-release-1.11/ for this link to work.

File: website/blog/2024-06-24-release-1.11.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.11 Release! 🎉
+
+![Podman-desktop-1-11-hero](img/podman-desktop-release-1.11/banner.png)
+
+This release introduces:
+
+- **Light mode!**: Our most-requested feature is here! Try out our new experimental light mode in the settings.
```
Comment (deboer-tim) [inline website/blog/2024-06-24-release-1.11.md]: 'in the settings' -> 'in Settings'?

File: website/blog/2024-06-24-release-1.11.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+- **macOS Rosetta support for Apple Silicon**: Build AMD64 binaries at near-native speeds as ARM64 builds.
```
Comment (deboer-tim) [inline website/blog/2024-06-24-release-1.11.md]: Maybe drop the 'macOS'? Sounded repetitive with Apple Silicon to me.

The last part of 'Build AMD64 binaries at near-native speeds as ARM64 builds' was a little hard to parse. Maybe just 'Build AMD64 binaries at near-native speeds' or 'Build AMD64 binaries at almost the same speed as ARM64 binaries'?

File: website/blog/2024-06-24-release-1.11.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+- **Kubernetes improvements**: Check out our new Node and Volume pages for Kubernetes.
+- **Improved UI**: In addition to our light mode, you'll notice updates to our container listing page.
+- **Increased manifest support**: You'll now see images grouped together if built with a manifest.
```
Comment (deboer-tim) [inline website/blog/2024-06-24-release-1.11.md]: 'You'll now see images grouped together if built with a manifest.' -> simpler 'Images built with a manifest are now grouped together'?

File: website/blog/2024-06-24-release-1.11.md [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+
+---
+
+## Other Notable Enhancements
+
+We've added lots of features this release, here are some other highlights:
+
+- feat(PreferenceResource): adding light theme support for some preferences page by @axel7083 in [#7107](https://github.com/containers/podman-desktop/pull/7107)
+- feat: extensions should stay disabled after restart by @deboer-tim in [#7085](https://github.com/containers/podman-desktop/pull/7085)
+- feat: 1.10 release notes by @deboer-tim in [#6984](https://github.com/containers/podman-desktop/pull/6984)
```
Comment (deboer-tim) [inline website/blog/2024-06-24-release-1.11.md]: I usually remove any lines like this that are features, but 'internal', not really interesting to end users, or part of one of the enhancements we've already covered like light mode or kube improvements.

File: website/blog/2024-06-24-release-1.11.md [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+- fix: table child selection by @deboer-tim in [#7555](https://github.com/containers/podman-desktop/pull/7555)
+- fix: close watcher when FileSystemWatcher is disposed by @feloy in [#7503](https://github.com/containers/podman-desktop/pull/7503)
+- fix: table properly support undefined value result from rendermapping by @axel7083 in [#7723](https://github.com/containers/podman-desktop/pull/7723)
+- fix(UI): formpage missing content shadow by @axel7083 in [#7733](https://github.com/containers/podman-desktop/pull/7733)
+
+---
+
+## Other Chores
+
+We also made various other improvements and updates, including:
```
Comment (deboer-tim) [inline website/blog/2024-06-24-release-1.11.md]: Chores are usually really minor, internal, or setup for one of the features above. IMHO we should just say the total number of issues resolved or have a link if people are really interested in one of the previous sections, no details. This just bloats the release notes for something nobody will read.

File: website/blog/2024-06-24-release-1.11.md [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+
+![kubernetes enhancements](img/podman-desktop-release-1.11/pvc.png)
+
+### User Interface Improvements
+
+Take a moment to appreciate the upgraded user interface of Podman Desktop. You'll find that the container listing section, among others, has received significant enhancements, elevating your overall user experience.
+
+### Enhanced Manifest Support
+
+We've made strides in how we handle manifest images. Now, when building a multi-arch image, manifest images are grouped together, simplifying the process and enhancing efficiency.
```
Comment (deboer-tim) [inline website/blog/2024-06-24-release-1.11.md]: 'Now, when building a multi-arch image, manifest images are grouped together, simplifying the process and enhancing efficiency.' is a bit long with multiple commas. Maybe  'Multi-arch images are now grouped under the manifest, simplifying the process and enhancing efficiency.'?

File: website/blog/2024-06-24-release-1.11.md [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+
+<!--truncate-->
+
+---
+
+## Release Details
+
+### Light Mode 💡
+
+We're excited to announce the arrival of our most-requested feature.. light mode! You can now enjoy a brighter interface by toggling the light mode option in the settings section. Please note, this feature is currently experimental as we continue to optimize the rest of Podman Desktop for this new mode.
```
Comment (deboer-tim) [inline website/blog/2024-06-24-release-1.11.md]: Should we put 'Experimental' in the title, say 'almost here', or 'imminent arrival'? It feels a little like we're announcing it, then later mentioning that it's not quite done/fully here.

'..' looks like a typo, I'd go the full '...' or use '-' or ':' instead.

Settings should be capitalized, we often put the fa gear in too. Could also say the name of the setting or include a screenshot.

Maybe expand a little bit and mention this will help people with astigmatism or say thank you for your patience?

**Comment by cdrage** [comment]
Updated in a separate commit so you can see the changes, thanks for the thorough review! @deboer-tim

**Review by benoitf — COMMENTED** [review]
it is missing the image in static folder

(the one being used for the image on social media)

**Comment by cdrage** [comment]
> it is missing the image in static folder
> 
> (the one being used for the image on social media)

Added! Odd we have to add it to /static but I dont see it referenced anywhere. I'll add that to the release notes.

**Comment by benoitf** [comment]
@cdrage it's because the og:media link is generated from the static folder while the other images are resized/embedded into the docusaurus images (path is not known in advance)

**Review by benoitf — APPROVED** [review]

**Review by deboer-tim — APPROVED** [review]
